tests/dummy/app/templates/components/marker-layer-example.hbs

```diff
@@ -0,0 +1,4 @@
+<strong>Notes</strong>
+<p>
+  To customize your popup, pass a hash of values to the <code>popupOptions</code> property. The <a href="http://leafletjs.com/reference.html#popup">list of available options is available here.</a>
```

Comment:
You can use this link to link to the options directly: http://leafletjs.com/reference.html#popup-options